test/spec/modules/orbitsoftBidAdapter_spec.js

```diff
@@ -0,0 +1,358 @@
+describe('Orbitsoft Adapter tests', function () {
+  const expect = require('chai').expect;
+  const assert = require('chai').assert;
+  const adapter = require('modules/orbitsoftBidAdapter');
+  const bidmanager = require('src/bidmanager');
+  const adloader = require('src/adloader');
+  const CONSTANTS = require('src/constants.json');
+
+  const contentCallEndPoint = 'http://orbitsoft.com/ads/show/content?';
+  const jptCallEndPoint = 'http://orbitsoft.com/ads/show/hb?';
+
+  before(() => sinon.stub(document.body, 'appendChild'));
+  after(() => document.body.appendChild.restore());
+
+  describe('test orbitsoft callback response', function () {
+    it('should exist and be a function', function () {
+      expect(pbjs.handleOASCB).to.exist.and.to.be.a('function');
+    });
+
+    it('should add empty bid responses if no bids returned', function () {
+      let stubAddBidResponse = sinon.stub(bidmanager, 'addBidResponse');
+
+      let bidderRequest = {
+        bidderCode: 'orbitsoft',
+        bids: [
+          {
+            bidId: 'bidIdOrbitsoft1',
+            bidder: 'orbitsoft',
+            params: {
+              placementId: '16',
+              requestUrl: jptCallEndPoint
+            },
+            sizes: [[300, 250]],
+            placementCode: 'test-div-12345'
+          }
+        ]
+      };
+
+      // Empty bid response
+      let response = {
+        callback_uid: 'bidIdOrbitsoft1',
+        cpm: 0
+      };
+
+      pbjs._bidsRequested.push(bidderRequest);
```

Comment:
Shouldn't need to mess with Prebid's private vars.  This has the ability to affect other tests, plus it's a private var so it could possible it could change.